website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,317 @@
+For further information see: [CRD scope doc][crd-scope-doc].
+
+## Watching and Managing resources in all cluster (Default)
+
+A [Manager][ctrl-manager] is responsible for managing the controllers of the operator project. In the `main.go` you will 
+be able to find it. See:
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+})
+if err != nil {
+    setupLog.Error(err, "unable to start manager")
+    os.Exit(1)
+}
+...
+```
+
+**NOTE** The above example is the default code generated for the projects created with `operator-sdk init` which is 
+creating a new Manager for an operator `cluster-scoped`.
+
+## Watching and Managing resources in an Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to an specific `namespace` is required to inform it via 
+the `Namespace` attribute from [Options][ctrl-options]. See: 
+
+```go
+...
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    Namespace:          "operator-namespace",
+    MetricsBindAddress: metricsAddr,
+})
+...
+``` 
+
+## Watching and Managing resources in a set of Namespaces
+
+It is possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] attribute from 
+[Options][ctrl-options] to watch and manage resources only in a specific set of namespaces. See:
+
+```go
+...
+var namespaces []string // List of Namespaces
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a namespace that 
+is not in this List of `namespaces` then, its controller will not re-trigger its reconcile since the namespaced is NOT 
+part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate. 
+Also note that you may face performance issues when using this with a high number of namespaces.
+
+## Restricting Roles and permissions
+
+The `operator-scope` defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. 
+In this scenario, it would also be recommended to change the default 
+[Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] to be restrictive accordingly.
+
+In the projects, the permissions are specified in the directory `config/rbac/`. Note that there you will find `role.yaml` 
+and `role_binding.yaml` files which are to allow permissions across the cluster by default for the operator. 
```

Comment:
Addressed but with few changes since the idea here is lead the user to the `rbac` dir since they will find more files than just the role one which are relevant to these scenarios.

Reply:
Also, we need to speak about RoleBinding as well.